bmc-log-collector/bmc-http-client.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"crypto/tls"
+	//"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"net/http"
+	"os"
+	"time"
+)
+
+// タイムアウトが必要？
+// Get from Redfish on iDRAC webserver
+func bmcClient(url string) ([]byte, error) {
+	fmt.Println("client start to ", url)
+	tr := &http.Transport{
```

Comment:
Don't create a transport for each request.
Transports should be reused.

See: https://pkg.go.dev/net/http
> The [Client.Transport] typically has internal state (cached TCP connections), so Clients should be reused instead of created as needed. Clients are safe for concurrent use by multiple goroutines.